lua/hawtkeys/init.lua

```diff
@@ -120,11 +120,22 @@ function M.setup(config)
         group = auGroup,
     })
 
-    vim.api.nvim_create_user_command(
-        "Hawtkeys",
-        "lua require('hawtkeys.ui').show()",
-        {}
-    )
+    vim.api.nvim_create_user_command("Hawtkeys", function(args)
+        if args.fargs[#args.fargs]:lower() == "all" then
+            require("hawtkeys.ui").show_all()
+        elseif args.fargs[#args.fargs]:lower() == "dupes" then
+            require("hawtkeys.ui").show_dupes()
```

Comment:
```suggestion
        local cmd = args.fargs[1] and args.fargs[1]:gsub("^%s+", ""):gsub("%s$", ""):lower()
        if cmd == "all" then
            require("hawtkeys.ui").show_all()
        elseif cmd == "dupes" then
            require("hawtkeys.ui").show_dupes()
```

trailing / leading spaces